nextflow.config

```diff
@@ -62,6 +67,19 @@ params {
     variable_mods                   = 'Oxidation (M)'
     vcf_sheet                       = null
     annotate_ions                   = false
+    filter_mzml                     = false
+
+    // DeepLC settings
+    use_deeplc                      = false
+    deeplc_calibration_mode         = 'rt_bin'
```

Comment:
We could change this to 'idx_bin' since it outperformed the 'rt_bin' calibration in most cases.